test/unit/component.test.js

```diff
@@ -20,249 +22,262 @@ Component.registerComponent('TestComponent2', TestComponent2);
   Component.prototype.options_ = {
-    'example': {
-      'childOne': { 'foo': 'bar', 'asdf': 'fdsa' },
-      'childTwo': {},
-      'childThree': {}
+    example: {
+      childOne: { foo: 'bar', asdf: 'fdsa' },
+      childTwo: {},
+      childThree: {}
     }
   };
 
-  var comp = new Component(getFakePlayer(), {
-    'example': {
-      'childOne': { 'foo': 'baz', 'abc': '123' },
-      'childThree': false,
-      'childFour': {}
+  const comp = new Component(getFakePlayer(), {
+    example: {
+      childOne: { foo: 'baz', abc: '123' },
+      childThree: false,
+      childFour: {}
     }
   });
 
-  var mergedOptions = comp.options_;
-  var children = mergedOptions['example'];
+  const mergedOptions = comp.options_;
+  const children = mergedOptions.example;
 
-  strictEqual(children['childOne']['foo'], 'baz', 'value three levels deep overridden');
-  strictEqual(children['childOne']['asdf'], 'fdsa', 'value three levels deep maintained');
-  strictEqual(children['childOne']['abc'], '123', 'value three levels deep added');
-  ok(children['childTwo'], 'object two levels deep maintained');
-  strictEqual(children['childThree'], false, 'object two levels deep removed');
-  ok(children['childFour'], 'object two levels deep added');
+  QUnit.strictEqual(children.childOne.foo, 'baz', 'value three levels deep overridden');
+  QUnit.strictEqual(children.childOne.asdf, 'fdsa', 'value three levels deep maintained');
+  QUnit.strictEqual(children.childOne.abc, '123', 'value three levels deep added');
+  QUnit.ok(children.childTwo, 'object two levels deep maintained');
+  QUnit.strictEqual(children.childThree, false, 'object two levels deep removed');
+  QUnit.ok(children.childFour, 'object two levels deep added');
 
-  strictEqual(Component.prototype.options_['example']['childOne']['foo'], 'bar', 'prototype options were not overridden');
+  QUnit.strictEqual(Component.prototype.options_.example.childOne.foo, 'bar',
+  'prototype options were not overridden');
 
   // Reset default component options to none
   Component.prototype.options_ = null;
 });
 
-test('should init child components from component options', function(){
-  let testComp = new TestComponent1(TestHelpers.makePlayer(), {
+QUnit.test('should init child components from component options', function() {
+  const testComp = new TestComponent1(TestHelpers.makePlayer(), {
     testComponent2: false,
     testComponent4: {}
   });
 
-  ok(!testComp.childNameIndex_.testComponent2, 'we do not have testComponent2');
-  ok(testComp.childNameIndex_.testComponent4, 'we have a testComponent4');
+  QUnit.ok(!testComp.childNameIndex_.testComponent2, 'we do not have testComponent2');
+  QUnit.ok(testComp.childNameIndex_.testComponent4, 'we have a testComponent4');
 });
 
-test('should allows setting child options at the parent options level', function(){
-  var parent, options;
+QUnit.test('should allows setting child options at the parent options level', function() {
+  let parent;
+  let options;
```

Comment:
Can we combine this declaration and the assignment on line 189? `let options = {`